resources/views/formfields/relationship.blade.php

```diff
@@ -29,8 +29,13 @@
 				<select class="form-control select2" name="{{ $options->column }}">
 					@php 
 						$model = app($options->model);
-	            		$query = $model::all();
-	            	@endphp
+						$query = $model::all();
+					@endphp
+
+					@if($row->required === 0)
+						<option value="">None</option>
```

Comment:
Would it be an idea to replace the static `None` with an automatic translation?

Reply:
There already a translation. `generic.none` iirc

Reply:
Good idea. I have just added it. I was glad to see "none" was already described in the generic translation files.